### ISSUE TITLE
Not available in ST3 package manager?

### ISSUE THREAD

**Issue by zakdances** [derekchiang/Sublime-CoffeeScript-Formatter#2]
This is an excellent plugin concept, but I can't seem to find it via the ST3 package manager. Does it support ST2 only?

**Comment by derekchiang** [comment]
Yes, it currently only supports ST2.  The main challenge here is that, by default, ST3 doesn't unzip a package before using it, as a result of which I can't directly assess JS files in my package (they are zipped).  However there should be a way to force ST3 to unzip a particular package.  Looking into it.

**Comment by alexgorbatchev** [comment]
:+1: for st3

**Comment by vorandrew** [comment]
:+1: for st3

**Comment by egmont1227** [comment]
+1 st3

**Comment by derekchiang** [comment]
Guys, I'm overwhelmed by exams.  Patches plz.

**Comment by ctf0** [comment]
@derekchiang if u dont want the package to be zipped and instead installed as a normal folder just add a `.no-sublime-package` file to the package check
https://packagecontrol.io/docs/customizing_packages

**Comment by altsanz** [comment]
+1 on st3! :)

**Comment by NeXTs** [comment]
+1 os ST3 :+1:

**Comment by etsms** [comment]
Any love?
